.code-samples.meilisearch.yaml

```diff
@@ -18,6 +18,17 @@ delete_documents_1: |-
 search_post_1: |-
 get_update_1: |-
 get_all_updates_1: |-
+  ```csharp
+  MeilisearchClient client = new MeilisearchClient("http://localhost:7700", "masterKey");
+  var index = client.Index("Movies");
+  
+  var updates = await index.GetAllUpdateStatus();
+  foreach (UpdateStatus update in updates)
+  {
+      Console.WriteLine(update.Status);
+  }
+  ```
+  [About this SDK](https://www.github.com/meilisearch/meilisearch-dotnet)
```

Comment:
Can you remove the `[About this SDK](https://www.github.com/meilisearch/meilisearch-dotnet)` line, except in the getting_started examples? :)

Reply:
Also, no need to surround your example with `csharp`, except for the getting started example.